etc/kayobe/kolla/passwords.yml

```diff
@@ -1,1010 +1,1010 @@
 $ANSIBLE_VAULT;1.1;AES256
```

Comment:
Did you find out what these changes are?

Reply:
A password was updated automatically during the build by the looks of it

Reply:
Which one?

Reply:
https://github.com/stackhpc/alaska-kayobe-config/blob/7edd6fddf80789a8ce677573e1734a0469979efb/etc/kayobe/kolla/passwords.yml

Reply:
Which password though?

Reply:
If it helps, I normally I do something like ```ansible-vault view passwords.yml > new.yml```, git stash, or checkout the original file (making a backup first), then ```ansible-vault view passwords.yml > old.yml``` and then ```diff new.yml old.yml``` to see which passwords have been changed.